src/components/input/input.js

```diff
@@ -322,7 +322,9 @@ function inputTextareaDirective($mdUtil, $window, $mdAria) {
             node.style.minHeight = null;
           }
 
-          var rows = Math.max(min_rows, Math.round(node.scrollHeight / lineHeight));
+          var rows = Math.min(min_rows, Math.round(node.scrollHeight / lineHeight));
+          node.style.height = lineHeight * rows + "px";
+
           node.setAttribute("rows", rows);
```

Comment:
No, if the attribute ```rows``` is not specified, the rows should get a dynamic value of rows, which auto grows and auto shrinks. If there is the ````rows``` attribute specified the textarea should not grow and shrink, it should stay fixed at the specified amount of rows